src/DSL.ts

```diff
@@ -0,0 +1,20 @@
+import { queryStrict } from './Query';
+import { taskEither } from 'fp-ts/lib/TaskEither';
+import { refetch } from './Strategy';
+
+export function param<A>() {
+  return queryStrict((a: A) => taskEither.of<void, A>(a), refetch);
+}
+
+export { queryShallow, queryStrict, queryJSON } from './Query';
```

Comment:
writing the docs I realized that IMHO one of there should just be called **query**, probably `queryShallow`. This would allow A much easier approach for new people and I also believe that would unclutter codebases with pedantic information

Reply:
Agreed, and also something that has come up already during previous usability tests. Not 100% on the choice though (the default could even be `queryStrict`), but you can assume to have a single exported `query` for docs purposes